fair_mango/metrics/metrics.py

```diff
@@ -63,12 +193,49 @@ def __call__(self) -> tuple[Sequence[str], list[dict]]:
         return targets, results
 
     def all_data(self) -> pd.Series:
-        """Compute selection rate for all the dataset
+        """Compute overall selection rate corresponding to the whole dataset.
 
         Returns
         -------
         pd.Series
-            the target name and the corresponding selection rate
+            The target name and the corresponding selection rate.
```

Comment:
not clear. Specify that the index is the column name, and the corresponding value is the selection rate. Why not use a dict instead?

Reply:
I updated the docstring as you suggested. I return a pandas series because I use pandas `.mean()` function which return  `pandas.Series`. I don't see any benefit of converting to a `dict`. It will be just one extra step.

Reply:
the doc string does not seem to be updated. You are right, pd.Series is fine since the usage of this function is mainly internal. The benefits of using dict is that everyone knows how it works

Reply:
You can see the updated docstring [here](https://github.com/datategy/Fair-Mango/pull/17/commits/f0a38802c63d03115c87094e1a33f692fef95ce6).